spec/spec_helper.rb

```diff
@@ -140,6 +140,18 @@ def run_message(message)
   end
 end
 
+# Include the Razor::Data item schemas + related schema helpers for the
+# api spec tests, specifically the /api/collection endpoint spec tests.
+# Note that we chose to put these in a separate file, because they take
+# up a lot of space without adding much additional clarity.
+require_relative 'lib/razor/spec/collection_schemas'
+RSpec.configure do |c|
+  # extend is necessary to use the helpers outside of an "it" block;
+  # include is to use the helpers inside an "it" block.
+  c.extend Razor::Spec::CollectionSchemas, api_spec: true
```

Comment:
What purpose does this `api_spec` serve?

Reply:
That's to limit including those helpers to only the api_spec tests. If you look at api_spec, you'll see the `api_spec: true` annotation next to the describe. Otherwise without it, those helpers will be included in all tests.

Will add a clarifying comment above.

Reply:
Are you concerned about a performance slowdown to having those helpers available in other tests?

Reply:
No, I just don't see a need to include them in other tests when they're solely meant for api_spec.